crypto/ledger/ledger_secp256k1.go

```diff
@@ -14,16 +14,19 @@ import (
 	"github.com/cosmos/cosmos-sdk/crypto/types"
 )
 
-// discoverLedger defines a function to be invoked at runtime for discovering
-// a connected Ledger device.
-var discoverLedger discoverLedgerFn
+// options stores the Ledger Options that can be used to customize Ledger usage
+var options Options
```

Comment:
is there a way to not use a private global? Or is this inherent to how the ledger is setup and used?

Reply:
The implementation of the Ledger functions depends on the options provided, so there must be some form of local state. Let me know if there is a way to improve it, I am open to any ideas.

Reply:
Could we use dependency injection here @kocubinski?